### PR TITLE
infoschema:  correct comments mistake.

### PR DESCRIPTION
**Pull request by louishust**
@zhexuany

### CODE REVIEW

File: infoschema/builder.go [hunk 1]
```diff
@@ -206,8 +206,8 @@ func (b *Builder) copySchemasMap(oldIS *infoSchema) {
 	}
 }
 
-// When a table in the database has changed, we should create a new schemaTables instance, then do modifications
-// on the new one. Because old schemaTables must be read-only.
+// copySchemaTables create a new schemaTables instance when a table in the database has changed.
```
Comment (shenli) [inline infoschema/builder.go]: create -> creates

File: infoschema/builder.go [hunk 2]
```diff
@@ -206,8 +206,8 @@ func (b *Builder) copySchemasMap(oldIS *infoSchema) {
...
+// Do modifications on the new one because old schemaTables must be read-only.
```
Comment (shenli) [inline infoschema/builder.go]: It also does modifications

File: infoschema/infoschema_test.go [hunk 3]
```diff
@@ -196,7 +196,7 @@ func checkApplyCreateNonExistsTableDoesNotPanic(c *C, txn kv.Transaction, builde
 	c.Assert(infoschema.ErrTableNotExists.Equal(err), IsTrue)
 }
 
-// Make sure it is safe to concurrently create handle on multiple stores.
+// TestConcurrent make sure it is safe to concurrently create handle on multiple stores.
```
Comment (shenli) [inline infoschema/infoschema_test.go]: make -> makes

File: infoschema/tables.go [hunk 4]
```diff
@@ -803,7 +803,7 @@ const (
 	uniqueKeyType  = "UNIQUE"
 )
 
-// See https://dev.mysql.com/doc/refman/5.7/en/table-constraints-table.html
+// dataForTableConstraints See https://dev.mysql.com/doc/refman/5.7/en/table-constraints-table.html
```
Comment (shenli) [inline infoschema/tables.go]: dataForTableConstraints constructs data for table information_schema.constraints.

**Comment by shenli** [comment]
LGTM @tiancaiamao @XuHuaiyu PTAL

File: infoschema/infoschema_test.go (hunk 3 above)
Comment (XuHuaiyu) [inline infoschema/infoschema_test.go]: s/make/makes

Comment (louishust) [inline infoschema/infoschema_test.go]: sorry for missing that

**Pull request merged**